ui/src/assets/branding.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+const Logo = () => (
```

Comment:
I have no clue what this thing now looks like. Can't we instead store SVGs as SVG files, and use Webpack's `svg-inline-loader`?

Reply:
Yes. It is a copy-paste from core-ui where I have some problem when transpiling "svg-inline-loader"...
But here: `import { ReactComponent as Logo } from './logo.svg';`  is enough

Reply:
Let's try that then? I prefer SVGs to be 'normal' files, otherwise it's very hard to edit them and such.